lmp/bb-config.sh

```diff
@@ -143,12 +143,10 @@ INHERIT += "archiver"
 COPYLEFT_RECIPE_TYPES = "target"
 ARCHIVER_MODE[src] = "original"
 ARCHIVER_MODE[diff] = "1"
-EOFEOF
 
-if [ $(ls ../sstate-cache | wc -l) -ne 0 ] ; then
-	status "Found existing sstate cache, using local copy"
-	echo 'SSTATE_MIRRORS = ""' >> conf/auto.conf
-fi
+# no sstate mirrors, using local copy
+SSTATE_MIRRORS = ""
+EOFEOF
 
```

Comment:
while we are looking at this, should we also consider trying this:

 https://github.com/doanac/ci-scripts/commit/9504b573260b91a4e5506328a255bc1eeeff4150 ?

Reply:
Sure, that would be better as I believe you already tested that path.

Mine was just to remove something that wasn't really being used.

Will close this one then, and let you push the other change.